test/_old/test_timeperiods.py

```diff
@@ -527,8 +527,11 @@ def test_issue_1385(self):
             invalid_tm = time.mktime(dt.timetuple())
             self.assertFalse(tp.is_time_valid(invalid_tm))
 
-
-
+    def test_timeperiod_multiple_monday(self):
```

Comment:
You leave this file in the *test/_old* directory ? Why not moving it to the *test* directory ???